grafanalib/cloudwatch.py

```diff
@@ -26,7 +26,8 @@ class CloudwatchMetricsTarget(object):
     :param period: Cloudwatch data period
     :param refId: target reference id
     :param region: Cloudwatch region
-    :param statistics: Cloudwatch mathematic statistic
+    :param statistics: Cloudwatch mathematic statistics
+    :param statistic: Cloudwatch mathematic statistic
```

Comment:
Please could you add a note about why there are 2 fields ( < v9 and > v9 for example) and the difference in formatting (list vs string)?

Reply:
Added a note to show `statistics` deprecated